plugins/__init__.py

```diff
@@ -641,7 +641,7 @@ def emit_relation(r):
                 r.get_name(),
             )
 
-    rel = [emit_relation(r) for r in paired]
+    rel = [emit_relation(r) for r in list(paired.keys())]
```

Comment:
```suggestion
    rel = [emit_relation(r) for r in paired.keys()]
```
Pretty sure this is redundant. The `.keys()` as well actually but that makes it more explicit so there is something to say for it.